.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+<!-- Thank you for contributing! -->
+
+### Before submitting the PR, please make sure you do the following
+
+- [ ] I have read the [contributing guidelines](https://github.com/vitejs/vite/blob/main/.github/contributing.md).
+- [ ] I have checked if there is already a issue, pr or discussion, so I don't create a duplicate
+- [ ] I provided a helpful description in this PR that addresses especially the **what** this PR is solving.
+      _Alternatively I referenced the issue with e.g. `fixes #123`_.
+- [ ] Ideally, I include also test that fails without this PR but passes with it.
```

Comment:
Consider not using first-person, and instead describe the action the person should take:

> - [ ] Read the [contributing guidelines](https://github.com/vitejs/vite/blob/main/.github/contributing.md).
> - [ ] Check that there isn't already a PR that solves the problem the same way to avoid creating a duplicate.
> - [ ] Provide a description in this PR that addresses what the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
> - [ ] Ideally, include relevant tests that fail without this PR but pass with it.

Also note that PRs that solve the same issue *different* ways should be valid - there might be a better approach to solving one problem.

Reply:
+1, good catch as this is also the contention in the issue template https://github.com/vitejs/vite/blob/main/.github/ISSUE_TEMPLATE/bug_report.md